StartDialog.gd

```diff
@@ -0,0 +1,9 @@
+extends Control
+
+func _ready() -> void:
+	if OS.get_name() == "X11" or OS.get_name() == "Server":
+		ProjectSettings.set_setting("display/window/size/borderless", false)
+		get_tree().change_scene("res://Main.tscn")
+	else:
+		ProjectSettings.set_setting("display/window/size/borderless", true)
+		get_tree().change_scene("res://Main.tscn")
```

Comment:
1. just a matter of style here, I'd move the changing of scene outside of the if/else block as it gets triggered regardless in the same way. 
2. since I can't test this I don't know how well this works, however I often made use of : [this](https://docs.godotengine.org/en/stable/classes/class_os.html#class-os-property-window-borderless) instead, as the project setting can stay the same, what matters is the window.

*As mentioned however this might work the same, if it does, then all good.*

Reply:
Using `OS.set_borderless_window()` is a lot cleaner and on my machine, it appears to work the same way.